conf/imperial.config

```diff
@@ -0,0 +1,37 @@
+//Profile config names for nf-core/configs
+
+params {
+  // Config Params
+  config_profile_description = 'Imperial College London - HPC Profile -- provided by nf-core/configs.'
+  config_profile_contact = 'Combiz Khozoie (c.khozoie@imperial.ac.uk)'
+  config_profile_url = 'https://www.imperial.ac.uk/admin-services/ict/self-service/research-support/rcs/'
+
+  // Resources
+  max_memory = 256.GB
+  max_cpus = 32
+  max_time = 72.h
+}
+
+executor {
+    $pbspro {
+	    queueSize = 50
```

Comment:
Mixture of 4 space and 2 space tabs in some of these files @combiz 😱

Reply:
:scream: Fixed in 17cf4e3 !